### ISSUE TITLE
[i3Kornhe] moving floating windows is lagging when keys are held

### ISSUE THREAD

**Issue by budRich** [budlabs/i3ass#96]
or to be honest, tiled windows are also a bit laggy when moved. But it is worse with the floating ones, and it has probably to do with the agressive title_format updating stuff. 

I think it is worth revisiting (rewriting) i3Kornhe and figure out a way to see if a key is being held down when the script is about to terminate. If there is a key (not a modifier) repeat the last action. The *problem* is that we don't really know what key-bindings the user have set to execute i3Kornhe, but i think it's enough to just test for any non-modifier key being held and assume it is the same key that triggered the command initially. By doing something like this we will not need to query i3list and stuff for each movement and things should be a lot smoother.

brainstorm:
if it isn't too heavy we could do a dirty config parse and get the actual i3Kornhe bindings... since config is gettable from IPC, it shouldn't tax the startup more then ~20ms to do it. worth trying at least.

**Comment by budRich** [comment]
or what about this:  
```
(
  i3-msg -tm subscribe '["binding"]' >> /tmp/Korhne_que
) &

# kohrne program

[[ -f /tmp/Korhne_que ]] && do the que...
```

need a way to kill the subscription, but this might be a nice way to do this without external tools...

**Comment by budRich** [comment]
resolved with #112
